app/Repositories/CategoryRepository.php

```diff
@@ -104,10 +104,12 @@ public function paginate(?int $perPage = Constants::DEFAULT_PER_PAGE)
      * @throws Some_Exception_Class A description of the exception that can be thrown.
      * @return \Illuminate\Contracts\Pagination\LengthAwarePaginator The paginated products.
      */
-    public function findByOrgUuidAndPaginate(string $orgUuid, ?int $perPage = Constants::DEFAULT_PER_PAGE)
+    public function findByOrgUuidAndPaginate(string $orgUuid, ?int $perPage = Constants::DEFAULT_PER_PAGE, ?string $search)
```

Comment:
Bos pa update ko na din pala to. instead na hiwa-hiwalay na arg yung perPage and search, then pati yung additional orderBy and orderDirection is ireceive nalang natin sya as array para mas madali gamitin tong method na to.